token/js/src/extensions/metadataPointer/instructions.ts

```diff
@@ -0,0 +1,95 @@
+import { struct, u8 } from '@solana/buffer-layout';
+import { publicKey } from '@solana/buffer-layout-utils';
+import type { Signer } from '@solana/web3.js';
+import { PublicKey, TransactionInstruction } from '@solana/web3.js';
+import { TOKEN_2022_PROGRAM_ID, programSupportsExtensions } from '../../constants.js';
+import { TokenUnsupportedInstructionError } from '../../errors.js';
+import { TokenInstruction } from '../../instructions/types.js';
+import { addSigners } from '../../instructions/internal.js';
+
+export enum MetadataPointerInstruction {
+    Initialize = 0,
+    Update = 1,
+}
+
+export const initializeMetadataPointer = struct<{
```

Comment:
nit: Let's call this something with `*Encoder` or `*Data`, just so it's marked as a serialization const. `initializeMetadataPointerData`?

Reply:
Updated!